### ISSUE TITLE
False Positive for DL3028

### ISSUE THREAD

**Issue by alexives** [hadolint/hadolint#325]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
`RUN gem install bundler -v '2.0.2'` Throws no warnings

### Actual behavior
Specifying version with `-v` instead of `:` throws DL3028 warning.

### Steps to reproduce the behavior
1. Create a Dockerfile:
```Dockerfile
FROM ruby:2.6.3
RUN gem install bundler -v '2.0.2'
```
2. Run
```
hadolint Dockerfile
```
3. Result
```
Dockerfile:2 DL3028 Pin versions in gem install. Instead of `gem install <gem>` use `gem install <gem>:<version>`
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.17.0-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
FROM ruby:2.6.3
RUN gem install bundler -v '2.0.2'
```

**Additional environment details (OS, stack version, etc.)**
None

**Comment by lekspyl** [comment]
I have similar, if not the same issue. Dockerfile contains this line:
```
RUN gem install nokogiri:1.7.2 -- --use-system-libraries=true --with-xml2-include=/usr/local/include/libxml2 --with-xml2-lib=/usr/local/lib
```
which fails linter check. I guess there's simply an issue with line parsing in my case, and for @alexives seems that old version pinning syntax is unsupported.

**Comment by lorenzo** [comment]
Interesting, thanks for reporting. I'll take care of this soon

**Comment by alexives** [comment]
Thanks so much for fixing this @lorenzo 🎉
